log4js / smpt -> module 'smtp' not found

just tried to setup my config to use the new features; is this missing module sth. that should be pulled by npm?

```
/epl_128/src/node_modules/log4js/lib/log4js.js:205
            throw new Error("Problem reading log4js config " + util.inspect(co
                  ^
Error: Problem reading log4js config { appenders: 
   [ { type: 'console', makers: [Object] },
     { type: 'file',
       filename: '/var/log/etherpad-lite/pad_access.log',
       maxLogSize: 16384,
       backups: 3,
       category: 'access',
       makers: [Object] },
     { type: 'logLevelFilter',
       level: 'error',
       appender: [Object],
       makers: [Object] } ] }. Error was "Cannot find module 'smtp'" (Error: Cannot find module 'smtp'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
    at Object.loadAppender (/home/epl/epl_128/src/node_modules/log4js/lib/log4js.js:290:26)
    at Object.configure (/home/epl/epl_128/src/node_modules/log4js/lib/appenders/logLevelFilter.js:14:12)
    at /home/epl/epl_128/src/node_modules/log4js/lib/log4js.js:148:59
    at Array.forEach (native)
    at configureAppenders (/home/epl/epl_128/src/node_modules/log4js/lib/log4js.js:144:22)
    at configureOnceOff (/home/epl/epl_128/src/node_modules/log4js/lib/log4js.js:196:13))
    at configureOnceOff (/home/epl/epl_128/src/node_modules/log4js/lib/log4js.js:205:19)
    at Object.configure (/home/epl/epl_128/src/node_modules/log4js/lib/log4js.js:256:5)
    at Object.reloadSettings (/home/epl/epl_128/src/node/utils/Settings.js:189:10)
    at Object.<anonymous> (/home/epl/epl_128/src/node/utils/Settings.js:204:9)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
RESTART!
```

--- a/settings.json.template
+++ b/settings.json.template
@@ -123,7 +123,7 @@
       }*/
     /*
 	  , { "type": "logLevelFilter"
-	    , "level": "error" // filters out all log messages that have a lower or same level than "error"
+	    , "level": "error" // filters out all log messages that have a lower level than "error"
 	    , "appender":
 	      { "type": "smtp"
         , "subject": "An error occured in your EPL instance!"

